pkg/reconciler/knativeserving/knativeserving.go

```diff
@@ -121,6 +122,9 @@ func (r *Reconciler) transform(ctx context.Context, manifest *mf.Manifest, comp
 		ksc.AggregationRuleTransform(manifest.Client),
 	}
 	extra = append(extra, r.extension.Transformers(instance)...)
+	extra = append(extra, ingress.Transformers(ctx, instance)...)
+
+	*manifest = manifest.Filter(ingress.Filters(instance))
```

Comment:
I was expecting the filtering to be in its own function, independent of the transforms, even if it's a one-liner, as a model of how other API's might be staged.

Reply:
will do in the proper version, thanks!